spec/unit/module/git_spec.rb

```diff
@@ -268,6 +268,48 @@ def test_module(extra_opts, env=nil)
             end
           end
         end
+
+        context "when using default_branch_override" do
+          before(:each) do
+            allow(mock_repo).to receive(:resolve).with(mock_env.ref).and_return(nil)
+          end
+
+          context "and the default branch override is resolvable" do
+            it "uses the override" do
+              expect(mock_repo).to receive(:resolve).with('default_override').and_return('5566aabb')
+              mod = test_module({branch: :control_branch,
+                                 default_branch: 'default',
+                                 default_branch_override: 'default_override'},
+                                 mock_env)
+              expect(mod.properties).to include(expected: 'default_override')
+            end
+          end
+
+          context "when default branch override is not resolvable" do
```

Comment:
(nit, just to match the other context above)
```suggestion
          context "and the default branch override is not resolvable" do
```